README.md

```diff
@@ -63,6 +64,13 @@ optional arguments:
                         Ignore tag, or more correct, a digest, if it is
                         referenced multiple times. ATTENTION: the default if
                         False!
+  -u BASICAUTHUSER, --basicauth-user BASICAUTHUSER
+                        The username, if the registry is protected with basic
+                        auth
+  -pw BASICAUTHPW, --basicauth-pw BASICAUTHPW
```

Comment:
can you add an example in the readme, too?
And a small part how to run the tests locally.
And please add yourself to the authors part in the py file.